install.sh

```diff
@@ -88,23 +95,29 @@ fi
 
 # Load configuration file
 echo "Loading config file '$config'..."
-source $config
+if [ ! -e "$config" ]; then
+  echo "Configuration files does not exist"
+  exit 1
+fi
+
+# shellcheck source=conf/zamba.conf.example
+source "$config"
 
-source $PWD/src/$service/constants-service.conf
+# shellcheck source=src/zmb-standalone/constants-service.conf
+source "$PWD/src/$service/constants-service.conf"
 
 # CHeck is the newest template available, else download it.
 DEB_LOC=$(pveam list $LXC_TEMPLATE_STORAGE | grep $LXC_TEMPLATE_VERSION | cut -d'_' -f2)
 DEB_REP=$(pveam available --section system | grep $LXC_TEMPLATE_VERSION | cut -d'_' -f2)
 
-if [[ $DEB_LOC == $DEB_REP ]];
-then
-  echo "Newest Version of $LXC_TEMPLATE_VERSION $DEP_REP exists.";
+if [[ $DEB_LOC == "$DEB_REP" ]]; then
```

Comment:
Template download mechanism reworked since Proxmox changed image format, this code (Line 113 - 120) will break rework.